app/lib/concerns/versioning_helpers.rb

```diff
@@ -10,18 +10,18 @@ def major_version_number(version)
   end
 
   def minor_version_number(version)
-    Gem::Version.new(version).segments[1] || 0
+    Gem::Version.new(version).segments[1] || Gem::Version.new(production_version).segments[1]
   end
 
   def prerelease_suffix?(version)
     Gem::Version.new(version).segments[2].eql?(PRERELEASE_SUFFIX) || false
   end
 
   def released_versions
-    ordered_versions.reject { |version| prerelease?(version_number(version)) }
+    ordered_versions.reject { |version| prerelease?(full_version_number_from(version)) }
   end
 
-  def version_number(version)
+  def full_version_number_from(version)
```

Comment:
I'm not sure the `_from` suffix is necessary here, my 2p is we don't typically suffix parameterized methods like this. Not a blocker though.

Reply:
@carlosmartinez started with naming this `full_version_number` however when we were looking through https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/controllers/concerns/versioning.rb#L15 we spotted a very similar method name `full_api_version_number`. The suggestion to add `_from` was to ensure that we don't have any conflict or confusion between both. Happy to hear your thoughts and possibly a better name to avoid discrepancies